src/renderer/components/ListenDevices.js

```diff
@@ -8,20 +8,11 @@ const ListenDevices = () => {
   const dispatch = useDispatch();
   useEffect(() => {
     let sub;
-    let timeout;
     function syncDevices() {
       const devices = {};
       sub = command("listenDevices")().subscribe(
         ({ device, deviceModel, type }) => {
           if (device) {
-            clearTimeout(timeout);
```

Comment:
there was a comment on this section that has not been ported: 

`// when there are no more device after 5s debounce, we'll restart the listen`

right after  `if (device) {`

Reply:
i don't see the comment here, but i'm trying to remove this workaround. we'll have to test if we have problem in future.

Reply:
👍